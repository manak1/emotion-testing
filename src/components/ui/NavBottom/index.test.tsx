import { render } from "~/test/test-utils"
import * as stories from "./index.stories"
import { composeStories } from "@storybook/testing-react"

const { Default } = composeStories(stories)

describe("NavBottomテスト", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Default />)
    expect(asFragment()).toMatchSnapshot()
  })
})
