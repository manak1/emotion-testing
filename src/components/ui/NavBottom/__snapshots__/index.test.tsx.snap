// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBottomテスト snapshot 1`] = `
<DocumentFragment>
  <nav
    class="css-lg20g"
  >
    <div
      class="css-1y58jls-NavBottom"
    >
      <ul
        class="css-1c029c7"
      >
        <li
          class="css-1hxuc68"
        >
          <a
            class="css-eilro2"
            href="/"
          >
            <svg
              aria-hidden="true"
              aria-label="ホーム"
              class="css-p4649-EmotionIconBase ex0cdmw0"
              fill="currentColor"
              focusable="false"
              height="40"
              role="img"
              viewBox="0 0 24 24"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
              />
            </svg>
          </a>
        </li>
        <li
          class="css-1hxuc68"
        >
          <a
            class="css-eilro2"
            href="/search"
          >
            <svg
              aria-hidden="true"
              aria-label="検索"
              class="css-p4649-EmotionIconBase ex0cdmw0"
              fill="currentColor"
              focusable="false"
              height="40"
              role="img"
              viewBox="0 0 24 24"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M15.5 14h-.79l-.28-.27A6.471 6.471 0 0016 9.5 6.5 6.5 0 109.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </a>
        </li>
        <li
          class="css-1hxuc68"
        >
          <a
            class="css-eilro2"
            href="/ranking"
          >
            <svg
              aria-hidden="true"
              aria-label="ランキング"
              class="css-p4649-EmotionIconBase ex0cdmw0"
              fill="currentColor"
              focusable="false"
              height="40"
              role="img"
              viewBox="0 0 24 24"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="24"
                width="24"
              />
              <path
                d="M19 5h-2V3H7v2H5c-1.1 0-2 .9-2 2v1c0 2.55 1.92 4.63 4.39 4.94A5.01 5.01 0 0011 15.9V19H7v2h10v-2h-4v-3.1a5.01 5.01 0 003.61-2.96C19.08 12.63 21 10.55 21 8V7c0-1.1-.9-2-2-2zM5 8V7h2v3.82C5.84 10.4 5 9.3 5 8zm14 0c0 1.3-.84 2.4-2 2.82V7h2v1z"
              />
            </svg>
          </a>
        </li>
        <li
          class="css-1hxuc68"
        >
          <a
            class="css-eilro2"
            href="/tag"
          >
            <svg
              aria-hidden="true"
              aria-label="タグ"
              class="css-p4649-EmotionIconBase ex0cdmw0"
              fill="currentColor"
              focusable="false"
              height="40"
              role="img"
              viewBox="0 0 24 24"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M20 10V8h-4V4h-2v4h-4V4H8v4H4v2h4v4H4v2h4v4h2v-4h4v4h2v-4h4v-2h-4v-4h4zm-6 4h-4v-4h4v4z"
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
  </nav>
</DocumentFragment>
`;
